tests/fixtures/functions/function_zxy_query.sql

```diff
@@ -16,3 +16,14 @@ BEGIN
   RETURN mvt;
 END
 $$ LANGUAGE plpgsql IMMUTABLE STRICT PARALLEL SAFE;
+
+DO $do$ BEGIN
+    EXECUTE 'COMMENT ON FUNCTION public.function_zxy_query (INT4, INT4, INT4, JSON) IS $tj$' || $$
+    {
+      "tilejson": "3.0.0",
```

Comment:
Don't set the version or tiles fields here and above

Reply:
I tried before, It's required for `tilejson` crate.